Kerbero.NET/src/Infrastructure/ConfigureServices.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class ConfigureServices
+{
+    public static IServiceCollection AddInfrastructureServices(this IServiceCollection services, IConfiguration configuration)
```

Comment:
```suggestion
    public static IServiceCollection AddInfrastructureServices(this IServiceCollection)
```
you should retrieve IConfiguration form DI where you need, not passing it as a parameter here